### PR TITLE
Add OAuth setup instructions

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: _includes/manuals/1.13/5.1.8_oauth.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+Alternatively to basic authentication, limited OAuth authentication is supported
```
Comment (domcleal) [inline _includes/manuals/1.13/5.1.8_oauth.md]: It'd be worth specifying that this is OAuth 1.0 (IIRC).

File: _includes/manuals/1.13/5.1.8_oauth.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+Please note that this header is not signed in OAuth request so can be forged.
+Anyone with valid consumer key can impersonate any Foreman user.
+
+#### Request example
+
+Usually some OAuth client library is used to generate the request. An example of
+curl command can be found here to better understand how it works
+
+    $ curl
+      -X GET 'https://foreman.example.tst/api/architectures' \
```
Comment (domcleal) [inline _includes/manuals/1.13/5.1.8_oauth.md]: example.com is standard, matches other examples

File: _includes/manuals/1.13/5.1.8_oauth.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      -H 'Content-Type: application/json' \
+      -H 'ForemanApiVersion: 2' \
```
Comment (domcleal) [inline _includes/manuals/1.13/5.1.8_oauth.md]: Is this correct? It should be an Accept header per 5.1.6, or preferably specify the version in the URL to keep the example smaller.

File: _includes/manuals/1.13/5.1.8_oauth.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+Anyone with valid consumer key can impersonate any Foreman user.
+
+#### Request example
+
+Usually some OAuth client library is used to generate the request. An example of
+curl command can be found here to better understand how it works
+
+    $ curl
+      -X GET 'https://foreman.example.tst/api/architectures' \
+      -H 'Content-Type: application/json' \
```
Comment (domcleal) [inline _includes/manuals/1.13/5.1.8_oauth.md:23]: Not needed for GET, remove to keep it smaller.

**Review by domcleal — CHANGES_REQUESTED** [review]

File: _includes/manuals/1.13/5.1.8_oauth.md (hunk 3 above)
Comment (ares) [reply _includes/manuals/1.13/5.1.8_oauth.md]: works for me for ages as well as -H 'Accept: application/json,version=2;' will change to keep it consisten

**Comment by ares** [comment]
Thanks @domcleal, fixed all your comments.

**Review by domcleal — APPROVED** [review]

**Comment by domcleal** [comment]
Thanks @ares.

**Pull request merged**